### PR TITLE
revert back to unstacked nft cards

### LINKED ISSUE #7480: Undo Stack cards
undo PR #7269 

Token entity queries are not well implemented and cause the indexer to crash

ref: #7462

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #7480

## Screenshot 📸

- [x] My fix has changed UI

### <samp>🤖 Generated by Copilot at 2676612</samp>

This pull request removes the concept of stacks from the NFT gallery UI and backend. It simplifies the data structure and query for fetching NFTs with metadata, and updates the components that display and interact with NFTs accordingly. It also deletes or disables some unused or redundant code related to stacks.

### <samp>🤖 Generated by Copilot at 2676612</samp>

> _The stacks of NFTs are no more_
> _They were removed from the `useNft` store_
> _The `NeoNftCard` and `ItemsGridImage` too_
> _Were updated to use the new data view_
> _And `useItemsGrid` got a refactor and more_

### CODE REVIEW

File: components/items/ItemsGrid/useItemsGrid.ts [hunk 1]
```diff
@@ -221,6 +191,10 @@ export const updatePotentialNftsForListingCart = async (nfts: NFT[]) => {
     )
     .map((nft) => {
       const collectionId = nft.collection?.id ?? nft.collectionId ?? ''
+      console.log(
+        'collection floor price',
+        statsById[collectionId]?.collectionFloorPrice
+      )
```
Comment (roiLeo) [inline components/items/ItemsGrid/useItemsGrid.ts:197]:
```suggestion
```

**Review by roiLeo — APPROVED** [review]

**Review by preschian — APPROVED** [review]
I hope we can see this stacks again~

**Comment by daiagi** [comment]
>I hope we can see this stacks again~

we will :wink:

**Comment by vikiival** [comment]
Does not make sense imo.
Squids run on full power anyway.

Screenshot 2023-10-03 at 10 45 50

File: components/items/ItemsGrid/useItemsGrid.ts [hunk 2]
```diff
@@ -101,38 +92,17 @@ export function useFetchSearch({
...
-    const handleToken = (token: any) => {
-      return {
-        ...token.nfts[0],
-        name: extractBaseName(token.nfts[0].name),
-        count: token.nfts.length,
-        floorPrice: Math.min(
-          ...token.nfts.map((nft) => Number(nft.price))
-        ).toString(),
-        nfts: token.nfts,
-      }
```
Comment (vikiival) [inline components/items/ItemsGrid/useItemsGrid.ts:118]: tak can be simply done by resolver

**Comment by daiagi** [comment]
@vikiival 

thought you said yes to revert.
np. close it

**Comment by vikiival** [comment]
> thought you said yes to revert. np. close it

Yup, but loooks like @yangwao is in favour of revert

**Pull request merged**